### PR TITLE
BUGFIX: Remove button icon background

### LINKED ISSUE #814: Icons within buttons have background
image

### PR DESCRIPTION
**Pull request by mstruebing**
closes #814

### CODE REVIEW

File: packages/react-ui-components/src/Button/style.css [hunk 1]
```diff
@@ -36,10 +36,12 @@
     padding: 0 8px;
 }
 
-.btn--clean,
-.btn--clean i {
+.btn--clean {
     background-color: var(--brandColorsContrastDarker);
 }
+.btn--clean i {
+    background-color: 0;
```
Comment (dimaip) [inline packages/react-ui-components/src/Button/style.css]: Hey, what does `background-color: 0;` mean? Why is it necessary?

Comment (mstruebing) [inline packages/react-ui-components/src/Button/style.css]: You are right :+1: 
`background-color: 0` is invalid
transparent or something like rgba(0, 0, 0, 0) is more what I was looking for. I will add this later

Comment (dimaip) [inline packages/react-ui-components/src/Button/style.css]: sorry for knit-picking, but why is it even needed to set a transparent background? It should be transparent by default, no?!

Comment (mstruebing) [inline packages/react-ui-components/src/Button/style.css]: Yep, you are right, sorry. PR updated :)

**Comment by dimaip** [comment]
Thanks!

**Pull request merged**